vcpkg/concepts/features.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Understanding features in vcpkg
+description: This article explains the concept of features, their capabilities, and their expected contents in detail.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 02/21/2024
+ms.topic: concept-article
+---
+
+# Concept: features
+
+## Features
+
+Manifests can define additive sets of functionality, behavior, and dependencies called _features_.
+
+Each feature should be additive with other features: if a project provides features `A` and `B`, then it should build successfully with both `A` and `B` enabled together. Features should not be used to define alternatives. For example, features cannot be used to choose between multiple, exclusive graphics APIs since it is not possible to choose both. Features can depend on other features within same project if the project's manifest has a [`"name"`](../reference/vcpkg-json.md#name).
+
+The set of features available are defined by the [`"features"` field](../reference/vcpkg-json.md#features).
+
+### Example 1: Multiple file formats
+
+An image manipulation library, for instance, might support several different image types by depending on different sets of other libraries.
+
+```json
+{
+  "name": "my-image-lib",
+  "version": "0.1",
+  "features": {
+    "png": { "description": "Support PNG files", "dependencies": ["libpng"]},
+    "jpeg": { "description": "Support JPEG files", "dependencies": ["libjpeg-turbo"]},
+    "tiff": { "description": "Support TIFF files", "dependencies": ["libtiff"]},
+  }
+}
+```
+
+
+## Default features
+
+Default features are a set of features to be automatically activated if the top-level project does not explicitly request a build without them. Default features are intended to ensure a minimum level of functionality regardless of how complex and customizable the dependency graph of a project grows. 
+
+>![NOTE]
+> They are not intended to model "curation" or "suggestions".
```

Comment:
In my opinion, we should have a contributor guideline regarding default features.